2-mandatory/3-stocks.js

```diff
@@ -35,6 +35,14 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
 */
 function getAveragePrices(closingPricesForAllStocks) {
     // TODO
+    let currentStock = [];
```

Comment:
This looks good 😄 
A suggestion: you might be able to improve the readability of this code by creating a separate function which calculates the average for a single stock. Each function will be a little bit simpler, and the new function will have a name - so we can more easily see what it's doing.